python/pyspark/sql/connect/session.py

```diff
@@ -885,12 +885,13 @@ def create_conf(**kwargs: Any) -> SparkConf:
                     SparkContext.getOrCreate(create_conf(loadDefaults=True, _jvm=SparkContext._jvm))
                 )
 
-                # Lastly remove all static configurations that are not allowed to set in the regular
-                # Spark Connect session.
-                jvm = SparkContext._jvm
-                utl = jvm.org.apache.spark.sql.api.python.PythonSQLUtils  # type: ignore[union-attr]
-                for conf_set in utl.listStaticSQLConfigs():
-                    opts.pop(conf_set._1(), None)
+                # Lastly only keep runtime configurations because other configurations are
+                # disallowed to set in the regular Spark Connect session.
+                utl = SparkContext._jvm.PythonSQLUtils  # type: ignore[union-attr]
```

Comment:
`org.apache.spark.sql.api.python.PythonSQLUtils` is automatically imported as a shortcut